packages/bigtable/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/bigtable",
-  "version": "0.8.0",
+  "version": "0.9.0",
```

Comment:
This part will be done by `npm run publish-module`, so you can leave the version untouched here and in the others that have that edit made.